ui/src/app/machines/views/MachineDetails/MachineHeader/MachineHeader.tsx

```diff
@@ -81,13 +85,31 @@ const MachineHeader = ({
         </>,
         <>
           <span className="u-nudge-left--small">
-            <Icon
-              name={`power-${checkingPower ? "checking" : machine.power_state}`}
+            <i
```

Comment:
I've filed https://github.com/canonical-web-and-design/react-components/issues/329 so that we can add the additional classes to `Icon`